oak-run/src/test/java/org/apache/jackrabbit/oak/run/DataStoreCommandTest.java

```diff
@@ -322,6 +325,71 @@ public void testConsistencyVerbose() throws Exception {
         testConsistency(dump, data, true);
     }
 
+    @Test
+    public void testConsistencyVerboseRootPathNoMissing() throws Exception {
+        File dump = temporaryFolder.newFolder();
+        Data data = prepareData(storeFixture, blobFixture, 10, 0, 0);
+        storeFixture.close();
+        additionalParams += " --verboseRootPath /c1,/c2";
+
+
+        for (String id : data.idToPath.keySet()) {
+            if (data.idToPath.get(id).equals("/c1") || data.idToPath.get(id).equals("/c2")) {
+                data.addedSubset.add(id);
+            }
+        }
+        testConsistency(dump, data, true, true);
+    }
+
+    @Test
+    public void testConsistencyVerboseRootPathWithDeleted() throws Exception {
+        File dump = temporaryFolder.newFolder();
+        Data data = prepareData(storeFixture, blobFixture, 10, 4, 1);
+        storeFixture.close();
+        additionalParams += " --verboseRootPath /c1,/c2";
+
+
+        for (String id : data.idToPath.keySet()) {
+            if (data.idToPath.get(id).equals("/c1") || data.idToPath.get(id).equals("/c2")) {
+                data.addedSubset.add(id);
+            }
+        }
+        testConsistency(dump, data, true, true);
+    }
+
+    @Test
+    public void testConsistencyVerboseRootPath() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/gh/oak-indexing/jackrabbit-oak/pullRequest?prid=5026544)